.github/workflows/generate-from-configuration.yaml

```diff
@@ -0,0 +1,113 @@
+name: Generate GAPIC libraries from configuration
+on:
+  schedule:
+  - cron: '0 2 * * *' # nightly at 2 am UTC
+  push:
+    branches:
+    - feat/hermetic-pr-creation
+  workflow_dispatch:
+
+
+jobs:
+  generate-from-configuration:
+    runs-on: ubuntu-22.04
+    env:
+      branch_name: generate-libraries
```

Comment:
This could be a follow up PR. Can we make this a parameter so that we can test the Github action in a separate branch? Otherwise it would always override the current PR if it is not merged.

Reply:
> Otherwise it would always override the current PR if it is not merged

This is an intended behavior because we don't want a lot of open PRs in the repo.

This is the [description](https://docs.google.com/document/d/1JiCcG3X7lnxaJErKe0ES_JkyU7ECb40nf2Xez3gWvuo/edit?tab=t.0#bookmark=id.h17h2gfiy57f) of the workflow.

Reply:
In addition, can we change the branch name to include the branch name, something like `generate-libraries-main`? As we would need to support LTS and multiple Java versions in the future.

Reply:
> > Otherwise it would always override the current PR if it is not merged
> 
> This is an intended behavior because we don't want a lot of open PRs in the repo.
> 
> This is the [description](https://docs.google.com/document/d/1JiCcG3X7lnxaJErKe0ES_JkyU7ECb40nf2Xez3gWvuo/edit?tab=t.0#bookmark=id.h17h2gfiy57f) of the workflow.

I agree with this behavior. What I'm suggesting is to make it more flexible so that we can test the Github action without overriding the PR.

Reply:
Done.